pkg/database/migrations.go

```diff
@@ -1639,6 +1638,35 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
+			ID: "00066-AddVerCodeUUIDUniqueIndexe",
+			Migrate: func(tx *gorm.DB) error {
+				sqls := []string{
+					`DROP INDEX IF EXISTS idx_vercode_uuid`,
```

Comment:
this could be expensive - should be noted in release notes

Reply:
Added to the notes